kernel/exec.c

```diff
@@ -218,3 +221,161 @@ exec(const char *path, const char* const *argv)
   }
   return result;
 }
+
+int
+execve(const char *path, const char *argv[], const char *envp[]){
```

Comment:
Could you replace `execelf` function to accept `envp` and use it in this function?
Also, could you remove `exec` function?

Reply:
It is done.
I've pushed it.